CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
    * FIXED: Make combine_route_stats.py properly quote CSV output (best practice improvement) [#3328](https://github.com/valhalla/valhalla/pull/3328)
    * FIXED: Merge edge segment records in map matching properly so that resulting edge indices in trace_attributes are valid [#3280](https://github.com/valhalla/valhalla/pull/3280)
    * FIXED: Shape walking map matcher now sets correct edge candidates used in the match for origin and destination location [#3329](https://github.com/valhalla/valhalla/pull/3329)
+   * FIXED: Better hash function of GraphId [#3332](https://github.com/valhalla/valhalla/pull/3332)
```

Comment:
I'm not sure this should be considered a fix. Probably more of an enhancement but no need to change this change log entry

Reply:
Specifically in case of `robin_hood::unordered_map` this caused a bug (insertion to unordered_map threw an exception), that is why I considered this a bugfix.

Reply:
I couldn't remember where we used it with that container. Actually in most cases I expected we would send the value directly to the container and let the container worry about hashing it. In other words the container template type would be uint64_t